### ISSUE TITLE
HELP, please guide me step by step to fix it 

### ISSUE THREAD

**Issue by GeoAxx** [overviewer/Minecraft-Overviewer#1995]

**Comment by GeoAxx** [comment]
Lmao, fisrt time using Gibhub

**Comment by Hackaloken** [comment]
Did you follow this guide in the documentation? http://docs.overviewer.org/en/latest/running/#installing-the-textures

**Comment by drnixon** [comment]
Hard to parse what you are asking for. Did your render fail? Did your render work but some textures are missing? Are ALL textures missing? What platform are you on? What version of the overviewer are you using? What version of Minecraft are you trying to render? We can't tell. You won't get much help aside from a reference to the documentation when you provide no details. When in doubt, RTFM first, provide specifics second, and include any troubleshooting steps you've already tried. No one here wants to be unhelpful, but you have to frame your question in a way that doesn't leave us guessing what it is you actually need.
